airflow/providers/google/cloud/triggers/kubernetes_engine.py

```diff
@@ -305,19 +307,35 @@ async def run(self) -> AsyncIterator[TriggerEvent]:  # type: ignore[override]
         if self.get_logs or self.do_xcom_push:
             pod = await self.hook.get_pod(name=self.pod_name, namespace=self.pod_namespace)
         if self.do_xcom_push:
+            kubernetes_provider = ProvidersManager().providers["apache-airflow-providers-cncf-kubernetes"]
+            kubernetes_provider_name = kubernetes_provider.data["package-name"]
+            kubernetes_provider_version = kubernetes_provider.version
+            min_version = "8.0.1"
```

Comment:
```suggestion
            min_version = "8.4.1"
```
@gopidesupavan you need to update `min_version`, because `do_xcom_push` functionality firstly was released in `8.4.1` version.

Reply:
@MaksYermak have updated to 8.4.1 and updated the condition, could you please check the condition i used here . 

min_version = "8.4.1"
if parse_version(kubernetes_provider_version) < parse_version(min_version):

is this looks fine?

Reply:
Was confused in the release logs its mentioned under bugfix , thought this feature already released :)

Reply:
I think min version is 8.4.2?
This change relays on releasing new version of Kubernetes provider. 8.4.1 doesn't contain the pull_interval

Reply:
are you referring to poll_interval?, it is already present in previous versions, but not using in `GKEJobTrigger` when calling these functions `wait_until_container_started`, `wait_until_container_complete`, `wait_until_job_complete`. so i have updated. not sure for this we have to update min_version to 8.4.2 and the poll_interval is in google cloud provider.

Reply:
ah right. I was confused by the change to the K8s hook

Reply:
Will raise separate pr for docstring updates.